### DISCUSSION TITLE
RBAC - cannot create custom roles with filtering query

### DISCUSSION THREAD

**Discussion by brazorfajeje** [Security-Onion-Solutions/securityonion#13931] [Unsupported Versions]
I am trying to create a custom role according to https://docs.securityonion.net/en/2.4/rbac.html but the query part seems not loaded in elastic.
after `so-checkin` I noticed a the message:
`
{"role":{"created":false}}{"role":{"created":false}}{"error":{"root_cause":[{"type":"security_exception","reason":"current license is non-compliant for [field and document level security]","license.expired.feature":"field and document level security"}],"type":"security_exception","reason":"current license is non-compliant for [field and document level security]","license.expired.feature":"field and document level security"},"status":403}{"role":{"created":false}}{"role":{"created":false}}
`

is that feature limited to paid subscritpions? 
I think if so it should be added in documentation.

**Comment by jertel** [comment]
Elastic has their own license/subscription limitations that are out of the Security Onion team's control. If you look at their [subscription comparison page](https://www.elastic.co/subscriptions) it shows what's included in each tier.

We'll look into adjusting the docs to help clarify this.
